Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -173,10 +173,17 @@ void MenuBar::AddViewMenu()
 void MenuBar::AddHelpMenu()
 {
   QMenu* help_menu = addMenu(tr("Help"));
+  QAction* website = help_menu->addAction(tr("Website"));
+  connect(website, &QAction::triggered, this,
+          [=]() { QDesktopServices::openUrl(QUrl(QStringLiteral("https://dolphin-emu.org/"))); });
```

Comment:
That was not my question. I was asking if there was any reason to have a = capture list, when it doesn't look like anything captured is being used in the lambda.